beacon-chain/forkchoice/doubly-linked-tree/epbs_test.go

```diff
@@ -0,0 +1,69 @@
+package doublylinkedtree
+
+import (
+	"context"
+	"testing"
+
+	"github.com/prysmaticlabs/prysm/v5/testing/require"
+)
+
+func TestStore_Insert_PayloadContent(t *testing.T) {
```

Comment:
Can you also add a test that covers the following conditions when calling `insert`?
```go
	n, rootPresent := s.nodeByRoot[root]
	m, hashPresent := s.nodeByPayload[payloadHash]
	if rootPresent {
		if payloadHash == [32]byte{} {
			return n, nil
		}
		if hashPresent {
			return m, nil
		}
	}
```